### ISSUE TITLE
[a11y] next-route-announcer is flagged as missing aria-atomic="true" by accessibility checkers.

### ISSUE THREAD

**Issue by Otoris** [vercel/next.js#25908]
### What version of Next.js are you using?

10.2.1

### What version of Node.js are you using?

14.16.1

### What browser are you using?

Chrome

### What operating system are you using?

macOS

### How are you deploying your application?

Vercel

### Describe the Bug

Accessibility checkers are flagging the next-route-announcer for not having the aria-atomic="true" attribute (see screenshot)

![image](https://user-images.githubusercontent.com/360427/121245224-c78f9b80-c86d-11eb-9f42-bf358c8d6582.png)
![image](https://user-images.githubusercontent.com/360427/121245285-dbd39880-c86d-11eb-9a55-e1653b58e927.png)

This is caught in both WAVE and SiteImprove.

I don't know if this is technically incorrect implementation, but more so a creative use of this property to announce dynamic page changes as this code does look to pass the tests described in ARIA19 -> https://www.w3.org/TR/WCAG20-TECHS/ARIA19

This would just be a nice to have to pass all automated accessibility checks.

### Expected Behavior

I would expect next-route-announcer to pass most accessibility testing tools.

### To Reproduce

Reproduce this on any page with dynamic routing.

**Comment by davidluhr** [comment]
I believe adding `aria-atomic="true"` is unnecessary, as `aria-live="assertive"` is basically a superset of `role="alert"` and `aria-atomic="true"`.

For comparison, `aria-live="polite"` maps to `role="status"` and `aria-atomic="false"` (or omitting `aria-atomic` altogether).

I've tested the output from `<next-route-announcer>` in both NVDA (using Firefox in Windows) and VoiceOver (using Safari in MacOS), and the announcements work as expected. I would be interested in feedback from others though!

**Comment by flaval** [comment]
Sometimes, on Firefox NVDA on Windows, my page titles are weird…

This is what I have encountered :
![image](https://user-images.githubusercontent.com/20976043/148136175-69931d15-0639-4c68-97d2-78c0852f1b48.png)
The first cicrle is for my title `<AppName> - Interstitiel`, and the second one for `<AppName> - Recherche`, as you can see it miss some characters and it's like words have break returns.

Some other times, always on Chrome, all is normal :
![image](https://user-images.githubusercontent.com/20976043/148138036-4683078f-c989-4d8c-9bef-243a386dd6c7.png)

I suspect that the lack of `aria-atomic` attribute to be the reason of this behavior…
